client/my-sites/checkout/src/lib/pix-confirmation.tsx

```diff
@@ -109,24 +116,31 @@ export function PixConfirmation( {
 	isJetpackNotAtomic: boolean;
 } ) {
 	const translate = useTranslate();
+	const [ hasCopied, setHasCopied ] = useState( false );
+
+	const copyButtonRef = useCopyToClipboard( qrCode, () => {
```

Comment:
Im not entirely sure what the compose package does, but shouldn't we just define our own function here rather than importing one if this function simply copys text to the clipboard and updates a local state variable?

Reply:
`useCopyToClipboard` is supposed to do what we want, but in my testing here it does not. I didn't have time to debug that package so I just added my own hack to make it work for now, but eventually it would be nice to use the standard way to do it. You're right that we don't need the hook in that case, but it also sets up the `ref` required to activate the copy behavior and I didn't want to have to do that on my own.

Now that this is in a String Freeze, I'll take a look to see how complex it is to either fix `useCopyToClipboard` or make my own.